justfile

```diff
@@ -27,37 +27,32 @@ bench language="[rust|python]": (_run_lang language \
 # Auto-fix all clippy warnings.
 fix language="[rust|python]": (_run_lang language \
         "cargo clippy --all-targets --all-features --workspace --fix --allow-staged --allow-dirty" \
-        "poetry run ruff check --fix"
+        "uv run ruff check --fix"
     )
 
 # Format the code.
 format language="[rust|python]": (_run_lang language \
         "cargo fmt" \
-        "poetry run ruff format"
+        "uv run ruff format"
     )
 
 # Generate a test coverage report.
 coverage language="[rust|python]": (_run_lang language \
         "cargo llvm-cov --lcov > lcov.info" \
-        "poetry run pytest --cov=./ --cov-report=html"
+        "uv run pytest --cov=./ --cov-report=html"
     )
 
 # Run unsoundness checks using miri
 miri:
     PROPTEST_DISABLE_FAILURE_PERSISTENCE=true MIRIFLAGS='-Zmiri-env-forward=PROPTEST_DISABLE_FAILURE_PERSISTENCE' cargo +nightly miri test
 
-# Load a shell with all the dependencies installed
-shell:
-    poetry shell
-
```

Comment:
We could do `source .venv/bin/activate` here, but not sure if it's a good idea...

Reply:
Doesn't really work with how justfile runs its commands :/